Ports/AvailablePorts.md

```diff
@@ -174,7 +174,7 @@ Please make sure to keep this list up to date when adding and updating ports. :^
 | [`tinycc`](tinycc/)                    | Tiny C Compiler (TinyCC)                                        | dev                      | https://github.com/TinyCC/tinycc                                               |
 | [`tinyscheme`](tinyscheme/)            | TinyScheme Interpreter                                          | 1.42                     | https://sourceforge.net/projects/tinyscheme/                                   |
 | [`tr`](tr/)                            | tr (OpenBSD)                                                    | 6.7                      | https://github.com/ibara/libpuffy                                              |
-| [`vim`](vim/)                          | Vim                                                             | 8.2.2772                 | https://www.vim.org/                                                           |
+| [`vim`](vim/)                          | Vim                                                             | 8.2.3800                 | https://www.vim.org/                                                           |
```

Comment:
Please squash this commit into the previous one. We generally want every single commit in a PR to pass the CI checks, not just the final one.

To squash these commits:
1. First, remove the commits while keeping their contents intact:
```shell
git reset --soft HEAD~2
````
2. Then, create a new commit from them:
```shell
git commit
````

Reply:
Thank you, that's extremely useful knowledge!